### DISCUSSION TITLE
Feature Request: Manual mapping

### DISCUSSION THREAD

**Discussion by pinecones-sx** [Valkirie/HandheldCompanion#716] [Ideas]
Please add a feature to manually choose keys & buttons to map to or from.

For instance, when choosing what controller button will trigger an action, instead of requiring a user to press the corresponding button on the controller, allow them to CHOOSE the button from a list.

Likewise, when choosing what action to trigger from a button or keypress (either on mapping or actions), allow users to manually choose what buttons or keypresses to emulate.

It would also be helpful to allow an option for specific orders for key triggers -- such as holding trigger first, then pressing the A button.

Finally, please allow remapping to trigger all available OEM buttons, when they are present.

**Comment by CasperH2O** [comment]
@pinecones-sx HC does not have this feature for the hotkeys, but it does for the Profile Layout Remapping, you can choose a button (gamepad, keyboard or mouse) and some modifiers. OEM buttons can also be remapped this way.
